### ISSUE TITLE
Support for Immutable.js

### ISSUE THREAD

**Issue by adamalbrecht** [gaearon/redux-devtools#51]
Since upgrading to 1.0, support for Immutable.js has stopped working. It shows the raw immutable object rather than converting it to JS.

There is some discussion of this in #48

![screen shot 2015-08-12 at 10 13 57 am](https://cloud.githubusercontent.com/assets/180203/9226072/e81aac8a-40da-11e5-95b0-a59c3e5ca6d5.png)

**Comment by FredyC** [comment]
Ah, so you were faster than me to report this. Thanks :) As @gaearon mentioned, this should be solved with duck typing, perhaps checking if there is a `toJS` method and invoking it before passing for any checks.

**Comment by hugobessaa** [comment]
I hit the same problem, but I'm using `mori` for my data-structures. I think the DevTools component could receive `converter`, a function that would be applied as a map function for the whole state.

```
<DebugPanel top right bottom>
  <DevTools 
    store={store}
    converter={mori.toJs}
    monitor={LogMonitor} />
</DebugPanel>

**Comment by FredyC** [comment]
@hugobessaa I see that even mori implements iterator interface, so there is really no need for a converter. We just need someone willing to implement (see #66).

**Comment by hugobessaa** [comment]
Oh, I see. Nice.

**Comment by weltenwort** [comment]
For everyone who's waiting for that pull request to be merged, here's a small workaround via the `select` property of the `DevTools` component:
```jsx
let selectDevToolsState = (state = {}) => Immutable.fromJS(state).toJS();

<DebugPanel top right bottom key="debugPanel">
    <DevTools store={store} select={selectDevToolsState} monitor={LogMonitor} />
</DebugPanel>
```

It's a bit crude, but does the job for now.

**Comment by gaearon** [comment]
Should be fixed in 1.1.0, please verify.

**Comment by weltenwort** [comment]
:+1: works for me, thank you!
